### DISCUSSION TITLE
Modifying Predefined Dummy Data in Customized Anytype Android App for College Use

### DISCUSSION THREAD

**Discussion by tekorakle** [anyproto/contributors#96] [Anytype apps]
I did a complete check on the anytype-kotlin source code, and I was unable to find the predefined dummy data. I want to know what to do to remove them or replace them with something else. Thanks

**Comment by fuksman** [comment]
Hey @NkosiDlamini, embedded data comes with the `anytype-heart` library. Can you please elaborate on how you plan to use it in your college? Maybe we can address your requirements with some features that are currently in progress. You can contact me via email at fuksman@anytype.io or schedule a quick call using this link: [https://calendar.app.google/BKaDGSmDKde1gFnW8](https://calendar.app.google/BKaDGSmDKde1gFnW8).

**Reply by tekorakle** [reply]
How can I utilize the anytype-heart library to update data in anytype-kotlin? Our college intends to adopt Anytype as the primary knowledge base and as an LMS for fostering collaboration between students and teachers. We aim to produce tutorial videos tailored to our students' specific needs. However, we're struggling to locate the default dummy account in the app after installation. Since you mentioned that it's located in the heart library, could you guide us on how to replace it with our custom videos and content?

**Reply by fuksman** [reply]
You can find the embedded data at this location: [https://github.com/anyproto/anytype-heart/tree/200be590340b82173c5c60b845e136f899f6d6bc/util/builtinobjects/data](https://github.com/anyproto/anytype-heart/tree/200be590340b82173c5c60b845e136f899f6d6bc/util/builtinobjects/data).

Please note that with future releases you will be able to share your space content through the Gallery. This will allow students to install a space with university-related content easily. In the meantime, you can create the space with the desired content, export it in Any-Block format, and share it with the students. They can then import it on their end with just a few clicks.

**Reply by tekorakle** [reply]
@fuksman please I still have problem modifying the dummy data and I have a couple of questions to help me understand how to handle the modification better.

Question 1: I will need help to know how to change the dummy data in the Anytype via android studio. I don’t want my users to see dummy data, but an empty workspace.

Question 2: I will need a tutorial or documentation to better understand how to use protocol/main/proto/ and how it affect the AnyType installation

Question 3: If am using a self-host, Does it change the dummy data on the Anytype and what kind of database do I need to set up a self-host?
